modules/wildfly/src/main/modules/ogm/couchdb/module.xml

```diff
@@ -10,11 +10,12 @@
         <resource-root path="hibernate-ogm-couchdb-${project.version}.jar" />
     </resources>
     <dependencies>
-        <module name="org.hibernate" export="true" services="import" slot="${hibernate.ogm.module.slot}" />
+        <module name="org.hibernate" slot="${hibernate.ogm.module.slot}" />
         <module name="org.hibernate.hql" slot="${hibernate.hql.module.slot}" />
 
         <module name="javax.ws.rs.api" />
         <module name="javax.persistence.api" />
+        <module name="javax.transaction.api" />
```

Comment:
What's the reason for not leaving `<module name="javax.transaction.api" export="true" />` in core an instead removing it from all the other modules? Core exposes a JTA type on one of its APIs used by these other modules, so IMO it's in charge to make it available rather than putting this task to the modules using it. Both works, just curious about your reasoning.